operator/internal/manifests/service_monitor.go

```diff
@@ -148,7 +148,7 @@ func newServiceMonitor(namespace, serviceMonitorName string, labels labels.Set,
 	}
 }
 
-func configureServiceMonitorPKI(podSpec *corev1.PodSpec, serviceName string) error {
+func configureTLS(podSpec *corev1.PodSpec, serviceName string) error {
```

Comment:
I believe we need to move this function out of `service_monitor.go`

Reply:
Sure. Was thinking abt moving it to `config.go`, WDYT?